prow/github/client.go

```diff
@@ -2316,3 +2316,75 @@ func (s *reloadingTokenSource) Token() (*oauth2.Token, error) {
 		AccessToken: string(s.getToken()),
 	}, nil
 }
+
+// GetOrgProjects returns the list of projects in this org.
```

Comment:
Could you please add the documentation link to this comment

Reply:
I've added the documentation links for `GetOrgProjects` and `GetProjectColumns`.